distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -1323,6 +1323,12 @@ tcp:refreshinterval=250
 # The Netatmo refresh token (see http://dev.netatmo.com/doc/authentication/usercred)
 #netatmo:refreshtoken=
 
+# The Netatmo unit system where M=Metric system, US=US System (optional, defaults to M)
```

Comment:
Added description for US and Metric. If user needs more control, they can always create a rule to do this conversion.